app/Http/Controllers/API/UsersController.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace App\Http\Controllers\API;
+
+use App\Http\Controllers\API\BaseController as BaseController;
+use Illuminate\Http\Request;
+
+class UsersController extends BaseController
+{
+    public function newUser(Request $request){
```

Comment:
change function name to create. please refer to the image below
![image](https://github.com/user-attachments/assets/b068cb62-7ebc-4916-b36f-7635f2adcb72)